Replace central server by user interface or SaaS platform (#3909)
centreon/centreon-documentation: cloud/installation/deploy-poller.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> With Centreon Cloud, your user interface is already ready to use. All you have to do is to install pollers in your infrastructure, and that is very easy: just execute a script and then export the configuration, and there you are!

Author: ```suggestion
With Centreon Cloud, your SaaS platform is already ready to use. All you have to do is to install pollers in your infrastructure, and that is very easy: just execute a script and then export the configuration, and there you are!
```

3. By default, the hostname of the machine will become the name of the poller. If you want to change the machine's hostname, use the following command: `hostnamectl set-hostname your-hostname`.

4. [In the user interface](../getting-started/interface.md#accessing-the-central-servers-interface), click the arrow next to **Pollers** at the left of the header bar.

5. In the pop-up that appears, click **Copy install command**. The command is copied to your clipboard.

6. On the server that will become your poller, in your terminal, paste the install command and execute it (this should take about 10 minutes).

7. When this is finished, in the user interface, go to **Configuration > Pollers > Pollers**. The new poller appears in the list of pollers.

>    * The address in the **IP Address** column is that of the poller as seen by the platform.

Author: ```suggestion
   * The address in the **IP Address** column is that of the poller as seen by the SaaS platform.
```

   * The poller is not running yet (**No** in the **Is running?** column).

## Step 2: Exporting the configuration and restarting the poller

Export the configuration for this poller:

> 1. On the user interface, go to **Configuration > Pollers > Pollers**, then select the poller you have just created.

Reviewer: ```suggestion
1. In the user interface, go to **Configuration > Pollers > Pollers**, then select the poller you have just created.
```
